libqpdf/QPDFParser.cc

```diff
@@ -38,11 +38,343 @@ QPDFParser::parse(bool& empty, bool content_stream)
     std::shared_ptr<QPDFObject> object;
     bool set_offset = false;
 
-    std::vector<StackFrame> stack{{input, st_top}};
+//    std::vector<StackFrame> stack{{input, st_top}};
```

Comment:
Add to commit message:

The new method is temporarily an (almost) complete copy of parse, which is
temporarily (almost) unchanged.